package.json

```diff
@@ -94,14 +107,7 @@
     "stylelint-custom-processor-loader": "0.6.0",
     "stylelint-processor-styled-components": "1.6.0",
     "ts-jest": "23.10.4",
-    "tslint": "5.11.0",
-    "tslint-config-airbnb": "5.11.1",
```

Comment:
Are we missing something when dropping the tslint version, `tslint-config-airbnb`? Should we take [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb) to use?

Reply:
Might make sense to use the eslint version of this. I think the main reason for not using it was to align with designs system repo. Will have a look what it would mean to add this as addition to everything else we use.